docs/content/preview/api/ycql/dml_select.md

```diff
@@ -205,7 +211,7 @@ ycqlsh:example> SELECT * FROM employees WHERE department_id = 1 AND employee_id
              1 |           2 | Accounting |          Jane
 ```
 
-### Select with condition on a regular column, using WHERE clause
+### Select with condition on a regular column, using WHERE clause and ALLOW FILTERING options
 
 ```sql
 ycqlsh:example> SELECT * FROM employees WHERE department_id = 1 AND employee_name = 'John' ALLOW FILTERING;
```

Comment:
@aravind-nallan-yb @pkj415 

This query works even without ALLOW FILTERING. Did the semantics change?

```cql
ycqlsh:ybdemo> SELECT * FROM employees WHERE employee_name = 'John' ALLOW FILTERING;

 department_id | employee_id | dept_name  | employee_name
---------------+-------------+------------+---------------
             1 |           1 | Accounting |          John


```